build/assets/topologyupdater/05_Daemonset.yaml

```diff
@@ -28,7 +28,6 @@ spec:
             fieldRef:
               fieldPath: spec.nodeName
         image: $(NODE_FEATURE_DISCOVERY_IMAGE)
-        imagePullPolicy: Always
```

Comment:
This is a placeholder, the real value is set at 
https://github.com/openshift/cluster-nfd-operator/blob/master/controllers/nodefeaturediscovery_controls.go#L430

Reply:
Yep, I wasn't sure regarding the topology updater, will revert it.

Reply:
ping me once this is reverted , and I will lgtm and kick start the cherry picking

Reply:
Done, thanks for your help!